### PR TITLE
add isort, black and flake8 checkers

### LINKED ISSUE #35: Setup isort, black and flake8
* Create requirements.txt with these packages (use pip-tools for this) + pre-commit package
* Setup pre-commit hook to check for code style
* Setup some CI (Jenkins, CircleCI or whatever else)
* Add info about running code style checkers into README

### PR DESCRIPTION
**Pull request by Adman**
* create pre-commit hook config file
* add info to README
* create workflow build checkers
* closes #35

Signed-off-by: Adrian Matejov <ado.matejov@gmail.com>

### CODE REVIEW

**Comment by mrshu** [comment]
@Adman it would be cool if we could get this set up after the whole structure gets updated!

**Comment by Adman** [comment]
@mrshu The PR has been updated and is ready.

File: README.md [hunk 1]
```diff
@@ -45,3 +45,20 @@ Which allows for at least some automation. Further info can be found in the
...
+
+## Development
+
+We are open to contributions! Have a look at our [issues](https://github.com/RoboCupJuniorTC/rcj-soccer-sim/issues).
+Before you make a pull request, make sure the code is formatted
+with `black` and `isort`, and `flake8` issues are fixed.
+
+To do so, follow these steps:
+
+1. Create virtualenv `virtualenv -p python3 venv`
```
Comment (mrshu) [inline README.md:57]: @Adman should this use `venv` perhaps? (As in https://docs.python.org/3/library/venv.html)

File: pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        .git
+     |  .github
+     |  docs
+     |  venv
+     |  worlds
+     )/
+)
+'''
+
+[tool.isort]
```
Comment (mrshu) [inline pyproject.toml:15]: @Adman just out of curiosity, how did you come up with these?

File: .github/workflows/check.yml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    branches:
+      - master
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.x
```
Comment (mrshu) [inline .github/workflows/check.yml:16]: @Adman should we pin a specific version here? Like the version we've let everyone know we'll be using?

**Review by mrshu — APPROVED** [review]
Just a couple of notes -- thanks @Adman !

File: pyproject.toml (hunk 2 above)
Comment (Adman) [reply pyproject.toml:15]: @mrshu Corporate environment taught me.

File: README.md (hunk 1 above)
Comment (Adman) [reply README.md:57]: Yeah, let's use that.

File: .pre-commit-config.yml [hunk 4]
```diff
@@ -0,0 +1,21 @@
+default_language_version:
```
[the remark this answers is not in the document]
Comment (Adman) [reply .pre-commit-config.yml:1]: My intention is to include also running tests in pre-commit once we have some.

File: .github/workflows/check.yml (hunk 3 above)
Comment (Adman) [reply .github/workflows/check.yml]: Yeah, pinned to 3.8.

**Comment by Adman** [comment]
@mrshu Thanks for the comments, merging now.

**Pull request merged**